src/components/Organisms/MarketingPreferencesDS/MarketingPreferencesDS.style.js

```diff
@@ -75,24 +75,24 @@ const AssociatedFieldsName = styled.span`
   }
 `;
 
-const FormField = styled.div`${({ theme }) => css`
+const FormField = styled.div`${({ theme, isError }) => css`
   position: relative;
   margin-bottom: ${spacing('md')};
   width: 100%;
   display: flex;
   flex-direction: column;
   padding: ${spacing('m')};
-  border: 1px solid ${theme.color('grey')};;
-  border-radius: 10px;
   background-color: ${theme.color('grey_light')};
   transition: background-color 0.3s, color 0.3s;
-
-  &:hover {
-    background-color: ${theme.color('grey_medium')};
-  }
+  border-radius: 10px;
+  border: 1px solid ${theme.color('grey')};;
 
   &.selected {
-    background-color: ${theme.color('blue_donate')};
+    background-color: ${isError ? theme.color('red') : theme.color('blue_donate')};
+    &:hover {
+      background-color: ${isError ? theme.color('blue_donate') : theme.color('blue_donate')};
```

Comment:
Seems like you don't need this check since the outcome is the same in both scenarios?

This is obviously Curtis' call but it feels weird to me that the red bg-colour goes away on hover when its _still_ erroring? Would probs speed up the transition either way; it current feels like a weird interaction to me in generally, esp given how massive the wrapper is